### PR TITLE
[WIP] Fix typo, fix url to relative

### LINKED ISSUE #16: Make Heroku prevent-sleep optional
For example, I'm hosting chat-linker on my own server and I don't need any `prevent-sleep.js` functionality. It should be possible to completely disable that module through the configuration.

### PR DESCRIPTION
**Pull request by Newlifer**
Also fix #16

### CODE REVIEW

**Comment by Newlifer** [comment]
@ForNeVeR need review, pls

File: app.js [hunk 1]
```diff
@@ -21,4 +21,9 @@ bus.on('message', ({network, room, name, message}) => {
 });
 
 // pill for heroku
-require('./prevent-sleep.js');
+
+const config = require('../app-config');
+const to_use_prevent_sleep = config['use_prevent_sleep'];
```
Comment (ForNeVeR) [inline app.js]: Please let's use `snakeCase` in config and variable names.

File: app.js [hunk 2]
```diff
@@ -21,4 +21,9 @@ bus.on('message', ({network, room, name, message}) => {
...
+if (to_use_prevent_sleep == true) {
```
Comment (ForNeVeR) [inline app.js]: Let's drop the `== true` part: `if (usePreventSleep) { ... }`

File: sample-config.json [hunk 3]
```diff
@@ -27,5 +27,6 @@
       "username": "javascript_ru",
       "type": "supergroup"
     }
-  }
+  },
+  "use_prevent_sleep": true
```
Comment (ForNeVeR) [inline sample-config.json]: Also I suggest to name this as just `preventSleep`.

Comment (Newlifer) [inline sample-config.json]: Changed to `usePreventSleep`.

File: app.js [hunk 4]
```diff
@@ -21,4 +21,10 @@ bus.on('message', ({network, room, name, message}) => {
 });
 
 // pill for heroku
-require('./prevent-sleep.js');
+
+const config = require('./app-config');
+
+const usePreventSleep = config.use_prevent_sleep;
```
Comment (ForNeVeR) [inline app.js]: Use ES6!
```js
const {usePreventSleep} = config;
```

**Comment by ForNeVeR** [comment]
@Newlifer as @jt3k reports that `usePreventSleep` is not working and not used anywhere, the proper solution now is to just *remove* the option and the corresponding code.

**Comment by jt3k** [comment]
yep! not working now :(

**Comment by jt3k** [comment]
i close PR. thanx for u work!

**Comment by ForNeVeR** [comment]
Alright!

**Pull request closed without merging**